cerberus/cerberus.py

```diff
@@ -701,6 +722,13 @@ def _validate_allof(self, allof, field, value):
         if not valid:
             self._error(field, errorstack)
 
+    def _validate_rename(self, rename, field, value):
+        if isinstance(rename, _str_type):
```

Comment:
I think schema validation will check the field value type only, the rename field is just a configuration option which should be a string since since its value will replace the original field name in the document.